droid-hal-device.inc

```diff
@@ -312,6 +312,11 @@ Summary: Straggler files for %{device} hw
 
 %package kernel-dtbo
 Summary: Placeholder for %{device} flashable dtbo if applicable
+# The device flashing info and scripts are in droid-configs
+Requires: oneshot
+Requires(post): droid-config-flashing
+%{_oneshot_requires_post}
+
```

Comment:
Nitpick, it's better when this new line (before %description) is added to the `rpm: Clean up rpm packaging` commit to make things very tidy:) (see other comment)